drivers/gles3/rasterizer_canvas_gles3.h

```diff
@@ -367,6 +367,8 @@ class RasterizerCanvasGLES3 : public RendererCanvasRender {
 
 	void set_time(double p_time);
 
+	virtual void set_debug_redraw(bool p_enabled, double p_time, const Color &p_color) override {}
```

Comment:
Might be a good idea to mention somewhere (in the toggle's tooltip?:thinking:) that currently it's not implemented for GLES3.

Reply:
Good thinking, maybe a warning would be sufficient for now

```suggestion
	virtual void set_debug_redraw(bool p_enabled, double p_time, const Color &p_color) override {
		if (p_enabled) {
			WARN_PRINT_ONCE("Debug CanvasItem Redraw is not available yet when using the GL Compatibility backend.);
		}
	}
```

Reply:
A warning makes sense, should be more informative/discoverable than a tooltip indeed. :+1: